Assignment6/Array.cs

```diff
@@ -1,10 +1,55 @@
 ﻿using System;
+using System.Collections;
 using System.Collections.Generic;
 
 namespace Assignment6
 {
-    public class Array<T>
+    public class Array<T> : ICollection<T>
     {
-        
+        public int Count => MyArray.Count;

```

Comment:
```suggestion
        public int Count => MyArray.Count;
        public int Capacity { get; }
```
Add a property called Capacity, per the assignment bullet point:

> - Add a Capacity property to return the capacity (the value specified in the constructor) of the collection.